client/src/featureform/dashboard_metadata.py

```diff
@@ -339,7 +339,7 @@ def source_variant(variantData):
                 except ValueError:
                     continue
             if variantRow["transformation"] == SourceType.DF_TRANSFORMATION.value:
-                definition = str(variantRow["definition"], "latin-1")
```

Comment:
what was this lol?

Reply:
I believe it was in issue with sqlite where the definition would generate invalid characters when serialized that couldnt be stored. Latin-1 is an encoding that doesnt use those characters